index.py

```diff
@@ -113,6 +119,16 @@ def get_runs(series, sample):
         rows = dqm.fetch_run_list(series, sample)
     return {'items': [r._asdict() for r in rows]}
 
+def get_ref(subsystem, data_run, series, sample):
+    config_dir = VARS['CONFIG']
+    with make_dqm() as dqm:
+        rows = dqm.fetch_run_list(series, sample)
+    ref_runs = []
+    for row in [r._asdict() for r in rows]:
+        ref_runs.append(row['name'])
+    refs = sql.fetch_refs(autodqm.cfg.get_subsystem(config_dir, subsystem), subsystem, data_run, ref_runs)
+    return {'items': refs}
```

Comment:
It's a bit hard to figure out what `refs` is here, but I think I've got it. From an api perspective, `items` should be an *array* of items (whatever those are). Maybe instead return 
```{'items': refs['ref_data'], 'candidates': refs['ref_cands']}```
If refs['ref_data'] isn't an array I think it should be.